content/client-server-api/modules/spaces.md

```diff
@@ -19,10 +19,10 @@ go a step further and explicitly ignore notification counts on space-rooms.
 
 Membership of a space is defined and controlled by the existing mechanisms which
 govern a room: [`m.room.member`](#mroommember), [`m.room.history_visibility`](#mroomhistory_visibility),
-and [`m.room.join_rules`](#mroomjoin_rules). Public spaces are encouraged to have
-a similar setup to public rooms: `world_readable` history visibility, published
-canonical alias, and suitably public `join_rule`. Invites, including third-party
-invites, still work just as they do in normal rooms as well.
+and [`m.room.join_rules`](#mroomjoin_rules). Canonical aliases and invites, including
+third-party invites, still work just as they do in normal rooms as well. Furthermore,
+spaces can also be published in the [room directory](#room-directory) to make them
+discoverable.
```

Comment:
I removed the recommendation because the spec doesn't actually define a "public room". The closest thing is the `public_chat` preset on [`/createRoom`](https://spec.matrix.org/v1.13/client-server-api/#post_matrixclientv3createroom). This, however, uses a history visibility of `shared` rather than `world_readable` which increases the ambiguity further.

It seems to sufficient to me to explain that spaces are governed by the same mechanisms as rooms and, therefore, can be made "public" in several dimensions.